web/src/main/webapp/WEB-INF/data/data/resources/map/config-viewer.xml

```diff
@@ -10,7 +10,7 @@
   <ows-context:ResourceList>
     <ows-context:Layer name="{type=mapquest}"
                        group="Background layers"
-                       hidden="false"
+                       hidden="true"
```

Comment:
Yes but in this PR, the search map layers are the same as the viewer map ones.
I don't think Stamen is a good layer for the viewer..